conda-recipe/run_test.sh

```diff
@@ -60,16 +70,16 @@ return_code=$(($return_code + $?))
 pytest --verbose -s "${sklex_root}/tests" $@ $(json_report_name legacy)
 return_code=$(($return_code + $?))
 
-pytest --verbose --pyargs daal4py $@ $(json_report_name daal4py)
+pytest --verbose --pyargs daal4py $@ $(json_report_name daal4py) "${COV_ARGS[@]}"
 return_code=$(($return_code + $?))
 
-pytest --verbose --pyargs sklearnex $@ $(json_report_name sklearnex)
+pytest --verbose --pyargs sklearnex $@ $(json_report_name sklearnex) "${COV_ARGS[@]}"
 return_code=$(($return_code + $?))
 
-pytest --verbose --pyargs onedal $@ $(json_report_name onedal)
+pytest --verbose --pyargs onedal $@ $(json_report_name onedal) "${COV_ARGS[@]}"
 return_code=$(($return_code + $?))
 
-pytest --verbose -s "${sklex_root}/.ci/scripts/test_global_patch.py" $@ $(json_report_name global_patching)
+pytest --verbose -s ${sklex_root}/.ci/scripts/test_global_patch.py $@ $(json_report_name global_patching) "${COV_ARGS[@]}"
```

Comment:
I think without the quotes it'd end up failing if there spaces in the path.

Reply:
Oh shoot, that was definitely an oversight. I'll make sure the quotes are readded.